sql/sqlite/tables.sql

```diff
@@ -11,13 +12,24 @@ CREATE TABLE IF NOT EXISTS labki_pack_registry (
 
 -- labki_pack_pages: pages belonging to an installed pack
 CREATE TABLE IF NOT EXISTS labki_pack_pages (
-  pack_id TEXT NOT NULL,
+  pack_uid TEXT NOT NULL,
+  pack_id TEXT,
   page_title TEXT NOT NULL,
   page_namespace INTEGER NOT NULL,
   page_id INTEGER,
   last_rev_id INTEGER,
   content_hash TEXT,
-  PRIMARY KEY (pack_id, page_title)
+  PRIMARY KEY (pack_uid, page_title)
+);
+
+-- labki_page_mapping: remembers original page keys to final titles
+CREATE TABLE IF NOT EXISTS labki_page_mapping (
+  pack_uid TEXT NOT NULL,
+  pack_id TEXT,
+  page_key TEXT NOT NULL,
+  final_title TEXT NOT NULL,
+  created_at INTEGER,
+  PRIMARY KEY (pack_uid, page_key)
```

Comment:
this seems like it should be done in both the packs and pages tables - each pack has an optional prefix (?), and each page has an optional manual rename. then we avoid the possibility of having conflicting information. it also is a little awkward - the `page` table doesn't actually have the real page title, that's in another table.

we need foreign keys and uniqueness constraints for this table if it remains an independent table, since it shouldn't be possible to have multiple mappings for a single page